match/allegations_clusters_nopd.py

```diff
@@ -0,0 +1,234 @@
+       '6e28eae5c214351b589232d6928ba551, 306c539950e8c6aaaa2b09c0c269f461',
+       'cbaaca1a04cc637d7569562cfb963c46, e990a37a007c2497fdb452408727ecd4, ddcf5e1e1b386bfc2306c6aa7ff09cff',
+       'bff90b3018c8e850c821b3f99d0d549b, a1c4c12aba057b4b06344e7bcb2718ff',
+       '1716776a1a5e6f49ead3c49960d040ea, a397984cfbe60ca52d142fe8a8963765',
+       '92f12c3529615b297a2472b77961f625, d23be2ef4b9479ab490763147169b9ea',
+       'f28f3c111a3d0832f0ec96a768912efe, 21be5cb8257dd88e02878d204fda9a71',
+       'e990a37a007c2497fdb452408727ecd4, 3ba732f2bfa3a3bcc258d2b2730beac4',
+       '4a8dbb9f8a0b1ffd35f54173d1227805, 81497e98ef1bd38a24443516b2c5269c',
+       '223494ad0ae821a6beb4fece42ee28f2, 9115b10c2dd837f90e53110a7ac625d9',
+       '1716776a1a5e6f49ead3c49960d040ea, 5997b252fe9d770d72bdf825ac772fb5',
+       '74044e2aa0e759e6e59d1745a9d77030, 3ba732f2bfa3a3bcc258d2b2730beac4',
+       '6a06e1a470912593c51418d2def7e858, 92f12c3529615b297a2472b77961f625',
+       '281c01a1b8a07a56609c5b10a9e1c148, e5c6e1680c5e0d758cf5c6354f15a1c9',
+       'b0f7be3fc290d4062c05e884a085148d, 3076f52bf765c928f9105e985c6207a9',
+       'cbaaca1a04cc637d7569562cfb963c46, 00f192f666f2e6ebeb0ec04e948febbd',
+       'ccad9784b3188744dc3fbd71590b7583, 21a71be2b52dc3f610acd7cc348c853a',
+       '4e973c5307167419176ae304f303e5d2, 0651ff554b810356fe536026da09aa8e, da5ec1607a4cc8a44b4b7963426fba40',
+       'a8766923063b332e60028b7de346cbe0, 500cd5c133f4bbcec9b02ee3984d51ff, ff669cc4b5878bd9d115fd1b6ac6c380',
+       'a34b9fecb7c8e863b92e8aea8559cf51, 8e4d87ec7c60b8e685e15dd9ecea104c',
+       '615431bff4472554462a8b6445a18585, 7c45171deb67212cdc3749a8046006ef',
+       '129139c7ce91abf9b4e8783adcec04c6, c4960591ab54569bf512525d239c38b4',
+       '2fee4778b97743ff6af445802d35c030, d00965a97518f81a20c9be784b96553e',
+       '681c8fe326cc2ad5cb04506f52574132, 738dc5819c13a6a029139dfd4b962cea',
+       'c75e6823cc3de53f0c4129af057acb4a, 40e190ec5d4a63c4e397e77f1410d2b5',
+       'd601272d7877017e5959c8dc67750ffa, d21f42a3196ab533161a1c446fe6f4d5',
+       '2b34d88c75e211c79055e5df7de66ecb, 71d3366908bf2f3d28d9e47cbbbe97a7',
+       'a50b15e88522ca16ca4901b5363730cf, d71bf1cd90cdf47820f7be40b7e0df7a',
+       '95f0da851956b555a435f511e6c76350, 19682fdd177ff6872cbef6f96a91c423',
+       '38655452c1a964184caf910e5be0c8ee, cb2c3497b94007fb1429c501da3c77b2',
+       '9aea0cd1efec57fa5b03d20f4d55b4be, 1a15535764f3158a07a75f9212319559',
+       '91f8792e8e1c08e379508f254f06a306, df9420dfb0af507791c67c401ca236f6',
+       '8174b1169b18edec85b8197d9e9d10b5, 88c9b8e47b0a813c2843e6d60e5eac57']
+    for tuple in list_of_duplicates:
+        for row in uids:
+            if tuple in row and tuple not in list_of_duplicates:
+                print(row)
+    return df
+
+
+def clean():
+    df = pd.read_csv(data_file_path('raw/ipm/allegations_nopd_merged.csv'))\
+        .pipe(float_to_int_str, ['tracking_number', 'uid', 'uids'])\
```

Comment:
`float_to_int_str` should only be used with decimal strings e.g. "2020". Which I think none of the columns qualify.

Reply:
Thanks.